team/README.md

```diff
@@ -0,0 +1,17 @@
+# The team folder
+
+All members of the organizing team should add themselves to the webpage and
+this folder contains a file for each person describing themselves.
+
+## Adding a new member
+Follow these steps to add yourself:
+* Use the provided 'template.yaml' file and copy the file.
+* Rename the file to your name
```

Comment:
Suggest deleting line 8 and replacing 9 - 12 with:
* Create a copy of the 'template.yaml' file and rename the file to your name:
```
 cp template.yaml FirstName_LastName.yaml
```